pkgs/development/compilers/solc/shared_install.patch

```diff
@@ -0,0 +1,88 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Is this your patch or is there an external source for it?

Reply:
It's my patch to enable shared solidity libraries build and install.